tests/python_on_whales/components/dummy_compose.yml

```diff
@@ -24,8 +24,8 @@ services:
     image: alpine:latest
     command: sleep infinity
     environment:
-     - DD_API_KEY=__your_datadog_api_key_here__
-     - POSTGRES_HOST_AUTH_METHOD=trust
+      - DD_API_KEY=__your_datadog_api_key_here__

```

Comment:
I'll let those formatting changes slide, but I'll ask that in the future you avoid adding formatting changes when they are not necessary. I think your IDE did it automatically, maybe you should disable auto-formatting yaml in this project.

Reply:
Oh sorry, i did not notice that. Will disable for future contributions